inbm/dispatcher-agent/dispatcher/aota/application_command.py

```diff
@@ -135,27 +135,15 @@ def update(self) -> None:
             # Move driver to CentOS filesystem
             shutil.move(driver_path, driver_centos_path)
 
-            # Remove old driver
-            old_driver_name = self.identify_package(driver_path.split('/')[-1])
-            if not old_driver_name:
-                raise AotaError(
-                    f'AOTA Command Failed: Unsupported driver {driver_path.split("/")[-1]}')
-            uninstall_driver_cmd = DOCKER_CHROOT_PREFIX + \
-                f'/usr/bin/rpm -e --nodeps {old_driver_name}'
-            out, err, code = PseudoShellRunner().run(uninstall_driver_cmd)
-            logger.debug(out)
-            # If old packages wasn't install on system, it will return error too.
-            if code != 0 and "is not installed" not in str(err):
-                raise AotaError(err)
-
             chroot_driver_path = driver_centos_path.replace("/host", "")
-            install_driver_cmd = DOCKER_CHROOT_PREFIX + f'/usr/bin/rpm -ivh {chroot_driver_path}'
+            install_driver_cmd = CHROOT_PREFIX + \
+                f'/usr/bin/rpm -Uvh --force {chroot_driver_path}'
```

Comment:
--force is a bad idea -- usually the dependencies are in the rpms for a good reason

Reply:
Without --force, the dispatcher will fail to install the driver that's older than current version. 
For example:
![image](https://user-images.githubusercontent.com/58315865/156078486-0fe0c051-3bfa-425d-b8c9-e17d73fff0f9.png)

@nmgaston Do we need to support backward driver update? My thought is the user might want to roll back the driver if the newly installed driver doesn't function well.

Reply:
@yengliong93 let's use --oldpackage for that use case instead of --force

Reply:
@gblewis1 RPM command updated with --oldpackage.